### DISCUSSION TITLE
How do I double open with the emulator?

### DISCUSSION THREAD

**Discussion by mechanic-Q** [AdriaGual/marvel-snap-bot#101] [Q&A]
I want to open two games and now only one port is recognized, what should I do?

**Comment by paperc07** [comment]
and you tried multi instance manager?

**Comment by paperc07** [comment]
an honestly now that I think about this it would be perfect cause then we could bot on a couple alt accounts 24/7

**Comment by AdriaGual** [comment]
Hmm what comes to my mind is that you would do an 'adb devices' when both emulators are on and you would see 2 ports (5555 and probably 5556), then you would need to run 2 scripts, one listening the port 5555 and the other 5556... But since now marvel snap can detect ADB... don't recommend using it on good accounts to be honest xD
